libs/community/langchain_community/vectorstores/azuresearch.py

```diff
@@ -236,14 +240,15 @@ def fmt_err(x: str) -> str:
 
 
 class AzureSearch(VectorStore):
-    """`Azure Cognitive Search` vector store."""
+    """`Azure AI Search` vector store."""
 
     def __init__(
         self,
-        azure_search_endpoint: str,
-        azure_search_key: str,
-        index_name: str,
-        embedding_function: Union[Callable, Embeddings],
+        azure_search_endpoint: str = "",
+        azure_search_key: str = "",
+        index_name: str = "",
+        embedding_function: Union[Callable, Embeddings] = lambda x: x,
+        azure_credential: Optional[Callable] = None,
```

Comment:
I'd put this down below `*,` in required keyword arguments in case users are passing these in positionally.

Reply:
Sounds good! I'll make sure that we can test things are working!